### DISCUSSION TITLE
Energy Management Tab

### DISCUSSION THREAD

**Discussion by mschut** [jobvk/Home-Assistant-Windcentrale#98] [Support] [answered]
Maybe this is a silly question, but I take the risk. I am using this plugin already for a while with great joy, especially the view and info in the Energy Managent Tab is great. I am however in doubt which is the best way to include the Windshares within this tab, as its not energy produced within my house, neither is it solar energy. My solution which I use so far is: 

Add the windshares in two ways: 

- As Solar energy,
- As Energy produced to the grid. 

In this way I feel I have the best insight in all the energy I produce (Wind and Solar), and how much I use. However, it still feels a bit made up. So, just checking, is this the way most of you use this info, or am I missing a better solution?  

![Schermafbeelding 2023-03-25 om 09 17 05](https://user-images.githubusercontent.com/11066663/227705944-751c974f-f4ee-4e7e-8ec5-59db03f9a2ae.png)
![Schermafbeelding 2023-03-25 om 09 17 17](https://user-images.githubusercontent.com/11066663/227705958-c7423b1d-419f-4a2b-80e8-fc4ed1ed0036.png)
![Schermafbeelding 2023-03-25 om 09 17 27](https://user-images.githubusercontent.com/11066663/227705966-1046eb1b-e1cf-4925-8afa-fa6ec1c42694.png)

**Comment by jobvk** [comment, marked as answer]
You can overwrite the energy dashboard to add a new graph just for the wind power. Example:
https://community.home-assistant.io/t/custom-energy-dashboard-how-to-place-energy-date-selection-card-in-header/406472

Here are some flow cards, all of them are available on HACS:
- https://github.com/flixlix/power-flow-card-plus ( I'm using this one )
- https://github.com/ulic75/power-flow-card
- https://github.com/flyrmyr/system-flow-card
- https://github.com/VeniVidiVici/givtcp-power-flow-card
